tools/sl4a_shell.py

```diff
@@ -48,12 +48,11 @@ def _start_services(self, console_env):
         console_env['ed'] = self._ad.ed
 
     def _get_banner(self, serial):
-        return """
-Connected to {}. Call methods against:
-    ad (android_device.AndroidDevice)
-    sl4a or s (SL4A)
-    ed (EventDispatcher)
-    """.format(serial)
+        return """Connected to {}. Call methods against:
```

Comment:
Can you run this and make sure it looks the same? I'm pretty sure the subsequent lines will end up hugely indented in the output this way.